x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/ForecastJobAction.java

```diff
@@ -116,9 +140,26 @@ public void setExpiresIn(TimeValue expiresIn) {
             }
         }
 
+        public void setMaxModelMemory(long numBytes) {
+            if (numBytes < MIN_MODEL_MEMORY) {
```

Comment:
sorry, me again: should we use 0 as `default`?

Reply:
@hendrikmuhs I am not sure what you mean `0`. I think allowing it to not be set at all covers the default value handling.

Reply:
true, as said its convenience: if you fire the API with a script which is reading the value from somewhere, you would not need to handle this special case in the script.

Not important, this is does not seem to be consistently used in the stack, or at least only for those parameters that change state.